cmd/data-node/commands/init.go

```diff
@@ -63,6 +65,25 @@ func (opts *InitCmd) Execute(args []string) error {
 	}
 
 	cfg := config.NewDefaultConfig()
+
+	if opts.Archive && opts.Lite {
+		return fmt.Errorf("specify either archive mode, lite mode - not both")
+	}
+
+	if opts.Archive {
+		for i, policy := range cfg.SQLStore.RetentionPolicies {
+			policy.DataRetentionPeriod = "forever"
+			cfg.SQLStore.RetentionPolicies[i] = policy
+		}
+	}
+
+	if opts.Lite {
+		for i, policy := range cfg.SQLStore.RetentionPolicies {
+			policy.DataRetentionPeriod = "1 day"
+			cfg.SQLStore.RetentionPolicies[i] = policy
+		}
+	}
```

Comment:
@candida-d we will want to document that somewhere when setting up a datanode.

Reply:
Noted, thanks @jeremyletang